aboutcitus/what_is_citus.rst

```diff
@@ -10,14 +10,20 @@ Citus extends the underlying database rather than forking it, which gives develo
 When to Use Citus
 -----------------
 
-Citus provides users real-time responsiveness over large datasets, most commonly seen in rapidly growing event systems or with time series data. Example use cases include:
+There are two situations where Citus particularly excels. The first is supporting real-time queries over large datasets. Commonly these queries occur in rapidly growing event systems or systems with time series data. Example use cases include:
```

Comment:
High-level, let's distinguish these with two separate sections maybe? My eye is drawn to the bullet points, which currently only capture real-time use-cases.
So maybe something like:
Two high-level use-cases:
- Multi-tenant:
-- Use-cases:
...
- Real-time
-- Use-cases:
...
 
 What the relevant use-cases are for multi-tenant is an interesting topic, and we can chat with @saicitus to see what verticals and use-cases we've seen so far. 

Also, let's skip the discussion on pro's/cons of the two models here, and instead move it to the 'data-modeling' section. We can just focus on providing a high-level overview/use-cases for each model.